### PR TITLE
0.8.1

### PR DESCRIPTION
**Pull request by helfer**
Thanks to @DxCx for fixing a bug where queries over HTTP GET would not go through validation.

### CODE REVIEW

**Comment by helfer** [comment]
Oh, now I see what's going on. I thought it was my mistake, but Lerna is auto-updating dependencies. Well how nice of Lerna ... 🙄

File: packages/graphql-server-core/package.json [hunk 1]
```diff
@@ -25,15 +25,15 @@
   "homepage": "https://github.com/apollostack/graphql-server#readme",
   "devDependencies": {
     "@types/fibers": "0.0.29",
-    "@types/graphql": "^0.9.0",
-    "fibers": "^1.0.15",
-    "meteor-promise": "^0.8.2"
+    "@types/graphql": "^0.9.1",
+    "fibers": "^2.0.0",
```
Comment (helfer) [inline packages/graphql-server-core/package.json]: This line here is the problem. Fibers should not be updated.

**Comment by helfer** [comment]
Maybe we can downgrade to a version of lerna that doesn't do the auto-updating.

**Comment by DxCx** [comment]
@helfer does it happen even if you request specified version?

**Comment by helfer** [comment]
Well, now that's clearly a bug and not consistent with the lerna docs... Updating fibers=1.0.15 to 2.0.0

**Comment by helfer** [comment]
Ah, I found the issue. It has nothing to do with Lerna at all. It's the `prerelease` script in package.json. @DxCx what was the rationale for adding `check-updates` to the lerna release cycle?

**Comment by DxCx** [comment]
Ohh right, sorry i forgot about that, but yeah, back then we had some issue
after release and it happend because one of the inner packages was not
updated by greenkeeper.

Anyway, we decided together on this behavior after it happened, we just
both forgot about it :)

**Comment by helfer** [comment]
Haha, I guess we both forgot about it! Well I've removed it for now because it was causing problems with fibers. We'll need to find another way to keep dependencies updated if we can't pin a certain version. In any case, I think we should run the version updates manually so we can fix them by hand if necessary.

**Pull request merged**